lib/vsc/administration/slurm/sync.py

```diff
@@ -165,6 +166,24 @@ def create_add_account_command(account, parent, organisation, cluster):
 
     return CREATE_ACCOUNT_COMMAND
 
+def create_change_account_fairshare_command(account, cluster, fairshare):
+    CHANGE_ACCOUNT_FAIRSHARE_COMMAND = [
```

Comment:
i'm not a big fan of uppercase variable names that aren't (module/class) constants

Reply:
You did not complain before :p